src/udp/socket.rs

```diff
@@ -595,7 +595,18 @@ quick_error! {
     }
 }
 
-const MAX_TTL: u32 = 16;
+// This is the maximum possible TTL. TTL runners never exceed this TTL.
+// It could be possible to set this as high as 255, but some OSes could plausibly have restrictions
+// against setting TTLs that high. Plus, 128 is already a huge value.
+const MAX_TTL: u32 = 128;
+
+// This is the default TTL used on Linux. Other OSes use anything from 30 to 128, but 64 is the
+// most common and the median value.
+const SANE_DEFAULT_TTL: u32 = 64;
+
+// How many hops we expect it to take, at most, to reach the peer. The slowest TTL runner will
+// reach this value over the course of HOLE_PUNCH_DELAY_TOLERANCE.
```

Comment:
Why not use doc comments? If we decided to make these constants public, we wouldn't need to change anything :)

Reply:
Sounds sensible to me :). Fixed.